src/Broadway/EventStore/Exception/DuplicatePlayheadException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Broadway\EventStore\Exception;
+
+use Broadway\EventStore\EventStoreException;
+use Exception;
+
+class DuplicatePlayheadException extends EventStoreException
+{
+    public static function create(Exception $exception)
```

Comment:
Good idea, it is a transaction, so providing the complete transaction (uncommtted events) makes a lot of sense!